tests/system/presenter/base.py

```diff
@@ -11,7 +11,7 @@ def get_application(self) -> WSGIApplication:
 
     def request(
         self, presenter: str, data: Optional[Dict[str, Any]] = None
-    ) -> Tuple[int, Dict[str, Any]]:
+    ) -> Tuple[int, Any]:
```

Comment:
Why did you change this?

Reply:
The presenter returns a list instead of an object.